docker/README.md

```diff
@@ -87,6 +87,37 @@ To setup Trusted Clusters:
     tctl -c /root/go/src/github.com/gravitational/teleport/docker/two-auth.yaml create -f docker/two-tc.yaml
     ```
 
+#### Trusted Clusters with File Configuration
+
+1. Export the CA for both clusters:
```

Comment:
numbered lists don't look that good in the docs really, consider adding subtitles instead to separate sections

**Export keys*